lib/tasks/cache.rake

```diff
@@ -6,4 +6,11 @@ namespace :cache do
     end
     Rails.logger.info('Thesis model counters have been updated.')
   end
+
+  desc 'Reset counter cache associated with the transfer model'
+  task reset_transfer_counters: :environment do
+    Rails.logger.info('Transfer counters update is starting.')
+    Transfer.find_each(&:save)
```

Comment:
Yeah, I think this should work. Could you add a comment referencing the `before_save` hook in the transfer model, so our future selves remember why this is done slightly differently than the thesis task? I just recalled we should also update the [resetting counter cache](https://github.com/MITLibraries/thing/blob/main/README.md#resetting-counter-cache) section of the readme.